dockerplugin.py

```diff
@@ -267,9 +307,9 @@ def run(self):
                 # Reset failure count on successfull read from the stats API.
                 failures = 0
             except Exception, e:
-                collectd.warning('Error reading stats from {container}: {msg}'
-                                 .format(container=_c(self._container), msg=e))
-
+                log.warning('Error reading stats from {container}: {msg}'
+                            .format(container=_c(self._container), msg=e))
+                log.info(traceback.format_exc())
```

Comment:
You should be able to use `log.exception('Error reading stats...` here and you'll get the traceback automatically in the log message as well.